modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/ListAppsTabs.es.js

```diff
@@ -22,13 +21,15 @@ import useLazy from '../../hooks/useLazy.es';
 
 export default (props) => {
 	const {appsTabs, appsTabsKeys} = useContext(AppContext);
-	const {tab} = props.match.params;
+	const {tab = appsTabsKeys[0]} = props.match.params;
 
-	if (!tab || !appsTabsKeys.includes(tab)) {
-		const initialTabKey = appsTabsKeys[0] || 'standard';
+	useEffect(() => {
+		if (props.history.location.pathname === '/') {
+			props.history.replace(`${appsTabsKeys[0]}`);
```

Comment:
Hey, @gabrielpramos, what do you think about use `tab` instead of `appsTabsKeys[0]`?

Reply:
sure :D